### PR TITLE
Mobille menu feature

### PR DESCRIPTION
**Pull request by david-lafontant**
# Create the mobile menu feature
- Create a script file
- Update style.css
- Update index.html

- [x] When the user clicks (or taps) the hamburger button, the mobile menu appears.
- [x] When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- [x] When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- [x] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -439,8 +439,42 @@ textarea {
...
+    height: 100vh;
+    position: fixed;
+    flex-direction: column;
+    gap: 0.8rem;
+    z-index: 10;
+    top: 0;
+    right: 0;
+    left: 0;
+
+    /* transition: top 2s ease; */
```
Comment (carlos-ssh) [inline css/style.css:453]: # 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 
- ⏏︎ Remove `/* commented code; */` from your project.
- Add The project name to your `README` file.
Screen Shot 2021-12-13 at 16 35 28

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @david-lafontant, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (david-lafontant) [reply css/style.css:453]: Thank you for pointing this.

**Review by ShoiraTa — COMMENTED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi team,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**